zstacklib/zstacklib/utils/fileUploader1.py

```
@@ -0,0 +1,108 @@
+
+class UploadTask(object):
+    def __init__(self, taskUuid, installPath, dstPath, tmpPath):
```

Ruff (0.13.1)

Unused method argument: `dstPath`

(ARG002)